### PR TITLE
Jaw/v0.23.0

### PR DESCRIPTION
**Pull request by jwoertink**
This PR updates the actual website to use Lucky 0.23 and Crystal 0.35

### CODE REVIEW

**Comment by jwoertink** [comment]
> Unsupported shard.lock. It was likely generated from a newer version of Shards.

I'm not sure what that means 🤔  maybe it's because Gh Actions is caching shards?

**Comment by paulcsmith** [comment]
I think the shard.lock is broken because you updated the Dockerfile, but we're now using GH actions. So I think the GH action file needs to be updated and Dockerfile removed. I think that'll fix it!

File: shard.yml [hunk 1]
```diff
@@ -8,27 +8,29 @@ targets:
...
     github: icyleaf/markd
     branch: master
   cadmium_transliterator:
     github: cadmiumcr/transliterator
     branch: master
+
+development_dependencies:
+  lucky_flow:
+    github: luckyframework/lucky_flow
+    version: ~> 0.7.0
```
Comment (paulcsmith) [inline shard.yml]: Could you add a newline here?

File: src/pages/home/index_page.cr [hunk 2]
```diff
@@ -53,7 +53,7 @@ class Home::IndexPage
   private def render_feature_grid
     div class: "mx-auto container" do
       div class: "text-center lg:mx-5 md:text-left px-6 py-5 bg-white md:bg-transparent border-b border-grey-light md:border-none" do
-        mount Shared::FlashMessages.new(@context.flash)
+        m Shared::FlashMessages, @context.flash
```
Comment (paulcsmith) [inline src/pages/home/index_page.cr:56]: I like this new `m` syntax :D

**Review by paulcsmith — APPROVED** [review]
Thanks for doing this! LGTM!

**Pull request merged**